### DISCUSSION TITLE
Invalid Define Constraint

### DISCUSSION THREAD

**Discussion by Alcal** [ReactUnity/core#33] [Q&A] [answered]
Hi guys, greatly interested in the project. I'm getting the following error after importing from the git repo
`Invalid Define Constraint: "UNITY_EDITOR || !REACT_DISABLE_UITOOLKIT" at line 2 (Packages/com.reactunity.core/Runtime/Frameworks/UIToolkit/ReactUnity.UIToolkit.asmdef)`
Is there anything I'm missing?

**Comment by gkurt** [comment, marked as answer]
Which Unity version?

Old Unity versions don't support this feature. Latest 2019.4 LTS version is required.

**Reply by Alcal** [reply]
Ah that must be it Unity 2019.4.3f1

**Reply by gkurt** [reply]
Yep, probably old Unity versions don't support this feature. LTS version of 2019.4 supports it though. I am using 2019.4.29
